packages/core/src/alert/alert.element.ts

```diff
@@ -280,7 +280,11 @@ export class CdsAlert extends AlertMixinClass {
         </span>
         <span
           class="alert-content-wrapper"
-          cds-layout="${getAlertContentLayout('wrapper', this.alertGroupType, this.parentGroupHasPager)}"
+          cds-layout="horizontal wrap:none ${getAlertContentLayout(
+            'wrapper',
+            this.alertGroupType,
+            this.parentGroupHasPager
+          )}"
```

Comment:
The fixes for layouts were to prevent the alignments leaking into nested child layouts. So its more explicit and only applies to direct children of the layout `>`. Alerts was relying on the buggy behavior I accidentally introduced so this fixes it and required the layout to be more explicit. Tested all the alerts, alert groups and alert with placeholder pager.